internal/character/qingque/skill.go

```diff
@@ -0,0 +1,44 @@
+package qingque
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	Skill key.Modifier = "qingque-skill"
+)
+
+func init() {
+	modifier.Register(Skill, modifier.Config{
+		StatusType: model.StatusType_STATUS_BUFF,
+		Stacking:   modifier.ReplaceBySource,
+		MaxCount:   4,
+		Listeners: modifier.Listeners{
+			OnAdd: skillOnAdd,
+		},
+	})
+}
+func (c *char) Skill(target key.TargetID, state info.ActionState) {
+	c.engine.AddModifier(target, info.Modifier{
+		Name:   Skill,
+		Source: c.id,
+	})
+	c.engine.InsertAction(c.id)
+}
```

Comment:
* missing logic to draw 2 cards?
* after 2 draw, if has 4 of same suit, need to give QQ "hidden hand"/talent modifier?
* Missing E4/Autarky?